scripts/tuple_tree_generator/tuple_tree_generator/generators/python.py

```diff
@@ -158,6 +165,45 @@ def to_bool(self, input: bool):
             return 'True'
         return 'False'
 
+    def add_mixins(self, formatted_rendered_template: str):
+        # Custom unpraser that doesn't remove backslashes
+        class Unparser(ast._Unparser):
+            def visit_Constant(self, node):
+                if isinstance(node.value, str) and node.lineno < node.end_lineno:
+                    super()._write_str_avoiding_backslashes(node.value)
+                    return
+                return super().visit_Constant(node)
+
+        def _unparse(ast_node):
+            unparser = Unparser()
+            return unparser.visit(ast_node)
+
+        # Use the `ast` module to add the appropriate mixin classes to the dataclasses in `_generated.py`
+        template_parsed = ast.parse(formatted_rendered_template)
+        template_classes = [node for node in template_parsed.body if isinstance(node, ast.ClassDef)]
+
+        with open(Path(self.mixins), "r") as mixins:
+            mixins_parsed = ast.parse(mixins.read())
+            mixin_classes = [node for node in mixins_parsed.body if isinstance(node, ast.ClassDef)]
+            for template_class in template_classes:
+                skip = False
+                for base in template_class.bases:
+                    if base.id not in ("Type", "TypeDefinition", "StructBase", "AbstractStructBase"):
```

Comment:
I was looping all the classes in the `_generated.py` and I needed to filter only the classes which should have a `mixin`. Since I changed the logic to use a filter this is not relevant anymore.